src/functions-and-arrays.js

```diff
@@ -52,16 +118,45 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
-
+function uniquifyArray(arr) {
+  let noDuplicates = [];
+  if (arr.length === 0) {
+    return null;
+  } else if (arr === undefined) {
+    return null;
+  }
+  for (let i = 0; i < arr.length; i++) {
+    if (noDuplicates.indexOf(arr[i]) === -1) {
+      noDuplicates.push(arr[i]);
+    } else {
+      continue;
+    }
+  }
+  return noDuplicates;
+}
 
+const noDuplicates = uniquifyArray(wordsUnique);
+console.log(noDuplicates);
 
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
-function doesWordExist() {}
-
+function doesWordExist(arrOfWords, word) {
+  if (arrOfWords.length === 0) {
+    return null;
+  }
+  for (let i = 0; i < arrOfWords.length; i++) {
+    if (arrOfWords[i].indexOf(word) !== -1) {
+      return true;
+    } else {
+      continue;
+    }
```

Comment:
with if statements you do not need to use `continue`, also in previous iterations.
You are inside the `for loop`, so you are already iterating through each element of the array , and only exit the loop if you find exisitng word in this case